tests/console/check_system_info.pm

```diff
@@ -61,6 +61,8 @@ sub check_product {
             $product = $proname{leap};
         } elsif ($mypro eq "SLE_HPC") {
             $product = $proname{SLES};
+        } elsif (($mypro eq "SLES") && (get_var("CROSS_PRODUCT_MIGRATION", 'from_hpc'))) {
```

Comment:
Should /s/get_val/check_val/ , or will cause failure. https://openqa.suse.de/tests/13770345/#step/check_system_info/17

Reply:
I submit a PR to fix it:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18878